tools/audio/extraction/tuning.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
+# tuning.py
+# SPDX-FileCopyrightText: © 2024 ZeldaRET
+# SPDX-License-Identifier: CC0-1.0
+#
+#   Estimate (samplerate, basenote) from tuning
+#
+#   tuning = samplerate * 2 ** basenote
+#
+
+from typing import List, Tuple
+
+from util import f32, u32_to_f32, f32_to_u32
+
+# Mirrors gPitchFrequencies in audio driver source.
+# Indexed by z64 note numbers, g_pitch_frequencies[C4] = 1.0 (0x3F800000)
+# Converted to their IEEE-754 binary representation to avoid any string -> float parser trouble as we need exact values.
```

Comment:
Is this really an issue? It's not like string -> float parsing is nondeterministic, and there's already `f32()` to ensure the results are single precision

Reply:
it is https://discord.com/channels/688807550715560050/688851451014938642/1100197439413178498

Reply:
true, `g_pitch_frequencies` could be a sequence of floats and then `g_pitch_frequencies = tuple(map(f32, g_pitch_frequencies))`

Reply:
if we want to use the same literals as gPitchFrequencies (which only have a few significant digits), I think we'd need f32() because the closest double to the string might be different from the closest float to the string

Reply:
Oh of course, I'm stoopid